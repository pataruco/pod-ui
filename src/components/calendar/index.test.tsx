import { shallow } from 'enzyme';
import React from 'react';
import { CalendarComponent } from './index';

describe('CalendarComponent', () => {
  const defaultProps = {
    dates: {},
    dataIsLoaded: true,
  };

  it('renders', () => {
    expect(shallow(<CalendarComponent {...defaultProps} />)).toMatchSnapshot();
  });
});
