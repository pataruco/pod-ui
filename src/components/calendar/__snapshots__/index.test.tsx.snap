// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarComponent renders 1`] = `
<section>
  <Uncontrolled(Calendar)
    events={Object {}}
    onSelectEvent={[Function]}
    selectable={false}
    views={
      Array [
        "month",
      ]
    }
  />
</section>
`;
